source/_integrations/saj.markdown

```diff
@@ -21,42 +23,15 @@ The Wi-Fi module requires a username and password for authentication where the e
 
 ## Configuration
 
-To enable this sensor, add the following lines to your `configuration.yaml` file:
+1. In the **Configuration** > **Integrations** menu, click the `+ ADD INTEGRATION` button on the lower-right of the screen.
+2. Select **SAJ Solar Inverter** from the pop-up menu.
+3. Complete the dialog box and click `SUBMIT`. 
```

Comment:
```suggestion
{% include integrations/config_flow.md %}
```

Please note: `## Configuration` also needs to be removed in this case. (Cannot suggest that, as it is outside of the change set).

Reply:
Done, thanks